tokio-trace-fmt/src/default.rs

```diff
@@ -94,7 +94,12 @@ impl<'a> field::Visit for Recorder<'a> {
         if field.name() == "message" {
             let _ = write!(self.writer, "{:?}", value);
         } else {
-            let _ = write!(self.writer, "{}={:?}", field, value);
+            if field.name().starts_with("r#") {
```

Comment:
TIOLI/style: I think this might read slightly better if we flattened this logic:
```rust
if field.name() == "message" {
    let _ = write!(self.writer, "{:?}", value);
} else if field.name().starts_with("r#") {
    let field = field.name();
    let _ = write!(self.writer, "{}={:?}", &field[2..], value);
} else {
    let _ = write!(self.writer, "{}={:?}", field, value);
}
```

or even rewrote this as a match:
```rust
let _ = match field.name() {
    "message" => write!(self.writer, "{:?}", value),
    name if name.starts_with("r#") => write!(self.writer, "{}={:?}", &name[2..], value),
    name => write!(self.writer, "{}={:?}", name, value),
}
```

Reply:
That is much better!